service/storage_service.cc

```diff
@@ -1911,6 +1914,138 @@ future<> storage_service::decommission() {
     });
 }
 
+// Runs inside seastar::async context
+void storage_service::run_replace_ops() {
+    if (!db().local().get_replace_address()) {
```

Comment:
(replace_address doesn't belong in `database`, that shouldn't be aware of clustering)

Reply:
This is historical and derived from C*.

Reply:
Yes, I put the comment in parenthesis to indicate it's not on the patch.